<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-08I2GPS" name="com.yaswanth.ecms.testcase.Loginpagetest" tests="4" failures="1" timestamp="2019-12-16T15:26:27 IST" time="27.707" errors="0">
  <testcase name="invalidCredentials" time="0.000" classname="com.yaswanth.ecms.testcase.Loginpagetest">
    <skipped/>
  </testcase> <!-- invalidCredentials -->
  <testcase name="validateLogin" time="8.093" classname="com.yaswanth.ecms.testcase.Loginpagetest">
    <failure type="java.lang.AssertionError" message="expected [https://apollo-ecms.firebaseapp.com/admin-dashboard] but found [https://apollo-ecms.firebaseapp.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://apollo-ecms.firebaseapp.com/admin-dashboard] but found [https://apollo-ecms.firebaseapp.com/]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.yaswanth.ecms.testcase.Loginpagetest.validateLogin(Loginpagetest.java:63)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateLogin -->
  <testcase name="forgotpassword" time="0.000" classname="com.yaswanth.ecms.testcase.Loginpagetest">
    <skipped/>
  </testcase> <!-- forgotpassword -->
  <testcase name="browserpage" time="19.614" classname="com.yaswanth.ecms.testcase.Loginpagetest"/>
</testsuite> <!-- com.yaswanth.ecms.testcase.Loginpagetest -->
